src/java.desktop/share/classes/java/awt/color/ICC_ColorSpace.java

```diff
@@ -527,16 +527,16 @@ public float[] fromCIEXYZ(float[] colorvalue) {
                     ColorTransform[] transforms = new ColorTransform[2];
                     var xyz = (ICC_ColorSpace) getInstance(CS_CIEXYZ);
                     PCMM mdl = CMSManager.getModule();
-                    transforms[0] = mdl.createTransform(xyz.getProfile(),
-                                    ColorTransform.Any, ColorTransform.In);
                     try {
-                        transforms[1] = mdl.createTransform(thisProfile,
+                        transforms[0] = mdl.createTransform(xyz.getProfile(),
                                         ICC_Profile.icRelativeColorimetric,
-                                        ColorTransform.Out);
+                                        ColorTransform.In);
                     } catch (CMMException e) {
-                        transforms[1] = mdl.createTransform(thisProfile,
-                                        ColorTransform.Any, ColorTransform.Out);
+                        transforms[0] = mdl.createTransform(xyz.getProfile(),
+                                        ColorTransform.Any, ColorTransform.In);
                     }
```

Comment:
Note that this try/catch block is not needed, we never throw the exceptions here, but I follow this pattern for now. Will clean up it in the separate PR.

Reply:
Is this some holdover from earlier code ?